webview-ui/src/components/chat/chat-view/components/layout/WelcomeSection.tsx

```diff
@@ -17,19 +18,20 @@ export const WelcomeSection: React.FC<WelcomeSectionProps> = ({
 	showAnnouncement,
 	hideAnnouncement,
 	showHistoryView,
-	telemetrySetting,
 	version,
 	taskHistory,
 	shouldShowQuickWins,
 }) => {
-	const { lastDismissedInfoBannerVersion, lastDismissedCliBannerVersion } = useExtensionState()
+	const { lastDismissedInfoBannerVersion, lastDismissedCliBannerVersion, platform } = useExtensionState()
```

Comment:
```suggestion
	const { lastDismissedInfoBannerVersion, lastDismissedCliBannerVersion, platform } = useExtensionState()
```

We actually don't need to pass platform into extension state here because the built-in browser process has the platform argument we can use.